src/Packetery/Module/Order/Form.php

```diff
@@ -53,25 +63,38 @@ public function __construct( FormFactory $formFactory ) {
 	 * @return Forms\Form
 	 */
 	public function create(): Forms\Form {
-		$form = $this->formFactory->create();
+		$form         = $this->formFactory->create();
+		$unit         = $this->options->getDimensionsUnit();
+		$floatPattern = '^\d+(\.\d+)?$';
```

Comment:
Tohle si myslím není vůbec potřeba. Pokud to bude cm, stačí `addRule( $form::FLOAT` pokud mm `$form::INTEGER`

Reply:
Mas pravdu. Je to lepsi varianta. Predelano.